packages/action-bar/stories/index.ts

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { html, TemplateResult } from '@spectrum-web-components/base';
+import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+
+export const ActionBarWrapper = ({
+    open = false,
+    emphasized = false,
+    variant = 'sticky',
+    flexible = false,
+    scoped = false,
+} = {}): TemplateResult => {
+    if (scoped) {
+        return html`
+            <sp-action-bar-scoped
```

Comment:
This still bloats the story count and maintenance load. It would be better to use a wrapper with render property approach that would reduce the additional contexts for a developing author.

It would be better to be able to do something in the realm of:
```
export const Default = () => html`<sp-button>Test</sp-button>`;
export const DefaultScoped = () => scopedStory(Default, [['sp-button', ButtonScoped]]);
```
Then the scoped stories could in theory be managed in a completely mechanical way as we ramp up to supporting it across the whole library.

Reply:
I have created a common util in shared folder which defines a scopedStory which can be reused easily but it would require us to define all child element which we are using in the storybook. 
`export const DefaultScoped = () => scopedStory(Default, [['sp-button', ButtonScoped]]);`